scripts/write-presubmit-scheduler.js

```diff
@@ -19,6 +19,7 @@ const logDir = '/tmp/v1-pr-presubmit-logs';
  * The default path is stored in an environment variable because it references an internal-Google
  * location.
  */
+// eslint-disable-next-line no-process-env
```

Comment:
ignore this in the 'scripts' overrides.

Reply:
I actually like that rule and want to eventually make changes to get it passing. So I don't want to disable it that widely.